lib/configuration/variables/sources/env.js

```diff
@@ -14,6 +14,7 @@ module.exports = {
     address = ensureString(address, {
       Error: ServerlessError,
       errorMessage: 'Non-string address argument in variable "env" source: %v',
+      errorCode: 'INVALID_ENV_SOURCE_ADDRESS_ARGUMENT',
```

Comment:
I think we can strip `_ARGUMENT` postfix (and same in other cases, technically _address_ and _params_ are an alias for _argument_ term here, so it kind of duplicates information)